Directory.Packages.props

```diff
@@ -128,24 +128,24 @@
     <PackageVersion Include="OpenTelemetry.Instrumentation.GrpcNetClient" Version="1.8.0-beta.1" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.8.1" />
     <PackageVersion Include="OpenTelemetry.Instrumentation.AspNetCore" Version="1.8.1" />
-    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.8.0" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Runtime" Version="1.8.1" />
     <!-- build dependencies -->
     <PackageVersion Include="MicroBuild.Plugins.SwixBuild.Dotnet" Version="1.1.87-gba258badda" />
-    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.9.2" />
+    <PackageVersion Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="4.10.0" />
```

Comment:
I think I ask this every time (will write a note this time for myself) but just to triple check, this isn't going to mean that in order to use Aspire you need to use a newer version of VS, right @eerhardt?

Reply:
> this isn't going to mean that in order to use Aspire you need to use a newer version of VS

That is correct. Notice the comment above this: `<!-- build dependencies -->`. We only use this version in our JSON schema generator.